src/smart-components/role/add-role-new/add-permissions.js

```diff
@@ -190,11 +190,10 @@ const AddPermissionsTable = ({ selectedPermissions, setSelectedPermissions, isIn
   }, [permissions, baseRole]);
 
   const setCheckedItems = (newSelection) => {
-    setSelectedPermissions(
-      newSelection(selectedPermissions)
-        .map(({ uuid }) => ({ uuid }))
-        .filter(({ uuid }) => getResourceType(uuid)?.count > 0)
-    );
+    const newSelected = newSelection(selectedPermissions)
+      .map(({ uuid }) => ({ uuid }))
+      .filter(({ uuid }) => uuid.split(':')[0] !== 'cost-management' || getResourceType(uuid)?.count > 0);
```

Comment:
Surely there must be some other rule to identify non-interactive elements. How do we identify them in the table itself?